ml/src/main/scala/ai/h2o/sparkling/ml/params/H2OCommonParams.scala

```diff
@@ -58,6 +60,8 @@ trait H2OCommonParams extends Params with Logging {
   //
   setDefault(
     predictionCol -> "prediction",
+    detailedPredictionCol -> "detailed_prediction",
+    withDetailedPredictionCol -> false,
```

Comment:
Could you tell me a benefit of disabling `detailedPredictionCol`? Is it heavy to compute?

Reply:
It works in a way that we always get full prediction (with all details) and select just the most interesting information and put that into `prediction` column.

If this is set to false, we remove the column with detailed predictions. I would like to avoid confusing users with 2 new colums for predictions. This way always the `prediction` is generated, and if the option is set to true, the `prediction_details` is kept as well.

See the transform logic in H2OMOjOModel

Reply:
And couldn't we generate `prediction_details `  all the time? It would be upto user whether he/she wants read the value or not.

Reply:
My opinion is that I would like to optimize the case for the basic user -> not adding unnecessary column for them if they are not interested in that. Advanced users can ask for it.

Reply:
ok, no problem